bundle-test/pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <artifactId>java</artifactId>
         <groupId>com.lightstep.tracer</groupId>
-        <version>0.19.2</version>
```

Comment:
I see we need to update our version script to include this one here, so this is aligned with the rest of the artifacts version (probably caused after merging the PR to select the tracer collector at runtime). You might want to use `0.20.` for now (else, wait for my fix and have it merged upstream).

Reply:
I'll wait for your fix for this.

Reply:
Oh, it was fixed yesterday ;)

Reply:
Rebased!